File: app/src/main/java/com/minwook/mytriphistory/present/ContentsPresentImpl.java
package com.minwook.mytriphistory.present;

import android.content.Context;

import com.minwook.mytriphistory.DAO.Content;
import com.minwook.mytriphistory.model.ContentsModel;

public class ContentsPresentImpl implements ContentsPresent {

    private ContentsPresent.View view;
    private ContentsModel model;

    public ContentsPresentImpl(ContentsPresent.View view){
        this.view = view;
        model = new ContentsModel();
    }


    @Override
    public void loadListData() {
        view.response(model.dataLoad());
    }
}

File: app/src/main/java/com/minwook/mytriphistory/kotlin/present/ContentsPresentImpl.kt
package com.minwook.mytriphistory.kotlin.present

import com.minwook.mytriphistory.kotlin.model.ContentsModel

class ContentsPresentImpl(var view: ContentsPresent.View): ContentsPresent {
    private val model: ContentsModel = ContentsModel()

    override fun loadListData() {
        view.response(model.dataLoad())
    }
}